.github/workflows/dluhc-build-and-publish.yml

```diff
@@ -13,7 +13,7 @@ on:
       ]
 
 env:
-  VERSION: "0.1.105" # Manually increment this version when pushing to main
+  VERSION: "0.1.107" # Manually increment this version when pushing to main
```

Comment:
https://github.com/communitiesuk/digital-form-builder/pull/236/files

This comes after https://github.com/communitiesuk/digital-form-builder/pull/236 - might need to solve conflicts.